### ISSUE TITLE
[Bug] Error appears in browser console when scroll Data Table

### ISSUE THREAD

**Issue by shabalin** [keplergl/kepler.gl#588]
Describe the bug

`bundle.js:85 Uncaught TypeError: n._root.querySelector is not a function` appears multiple times in console when scroll in Data Preview Modal

To Reproduce

Go to https://kepler.gl/demo/earthquakes
Click on  the dataset name 
Modal window with dataset preview table is opened 
Scroll down, check dev console output.
Expected behavior
no errors should be present in the console

Desktop (please complete the following information):

OS: macOS Sierra Version 10.13.6
Browser Chrome Version 75.0.3770.100

--------------------------------
In my own build I can see the error is coming from react-data-table
```
data-table-modal.js:160 Uncaught TypeError: _this._root.querySelector is not a function
    at data-table-modal.js:160
```
root object is a Ref, not a DOM element

**Comment by heshan0131** [comment]
will reopen #562 to address this

**Comment by macrigiuseppe** [comment]
Fixed in #629
